changelog/934.doc.rst

```diff
@@ -0,0 +1 @@
+Added description to Hall_parameter signature and equation in docstrings.
```

Comment:
```suggestion
Added description to :func:`~plasmapy.formulary.parameters.Hall_parameter`
signature and equation in docstrings.
```

The will allow the `Hall_parameter` to be automatically linked in the changelog.

Reply:
Oh!  I didn't know you could do this.  Good to know!